components/directory-size-exporter/go.mod

```diff
@@ -0,0 +1,32 @@
+module github.com/kyma-project/kyma/components/directory-size-exporter
+
+go 1.18
+
+require github.com/stretchr/testify v1.8.0
```

Comment:
The dependencies look a bit weird. At least the Prometheus cloud should not be an indirect dependency. Maybe try to run `go mod tidy`. Usually modules have one `require` block with direct dependencies and one with indirect ones.